### DISCUSSION TITLE
getsysex.sh Original DX7 ROMs Commented Out - Why?

### DISCUSSION THREAD

**Discussion by blixabloxa** [probonopd/MiniDexed#687] [Q&A]
Hi All,

Is there any reason why the first 4 original DX7 ROMs are commented out in the getsysex.sh script? 

I can easily uncomment these, renumber the prefixes so there are no duplicate prefixes, and run the script again - but I was wondering if there was a technical reason why these 4 ROMs are not included in the script.

Note, that I already have these ROMs (and many others) saved, so I don't actually need to run the script - I just copy them into the SD card sysex/voice folder as required.

Thanks,

Enzo

     # wget -c "${DIR}"/rom1a.syx -O sysex/voice/000000_rom1a.syx
     # wget -c "${DIR}"/rom1b.syx -O sysex/voice/000001_rom1b.syx
     # wget -c "${DIR}"/rom2a.syx -O sysex/voice/000002_rom2a.syx
     # wget -c "${DIR}"/rom2b.syx -O sysex/voice/000003_rom2b.syx
     wget -c "${DIR}"/rom3a.syx -O sysex/voice/000001_rom3a.syx
     wget -c "${DIR}"/rom3b.syx -O sysex/voice/000002_rom3b.syx 
     wget -c "${DIR}"/rom4a.syx -O sysex/voice/000003_rom4a.syx
     wget -c "${DIR}"/rom4b.syx -O sysex/voice/000004_rom4b.syx

**Comment by probonopd** [comment]
I think ROM 1 and 2 were used in Japan whereas 3 and 4 were used in the rest of the world. You can easily remove the comments if you like.

**Reply by Kirtai** [reply]
IIRC ROM 1 and 2 were also used in europe.
I may be misremembering though.

**Reply by probonopd** [reply]
> Info from The Complete DX7 (Massey) book mentions that ROMs 1 and 2 were for Japanese DX7 models, and ROMs 2 and 3 were for the US versions.

Source: https://forum.vintagesynth.com/viewtopic.php?t=50221#p531193 - not sure whether The Complete DX7 (Massey) or the post are correct, but that is probably what I had read.
